enforcer-parent/enforcer/src/main/java/org/wso2/choreo/connect/enforcer/security/jwt/APIKeyUtils.java

```diff
@@ -162,6 +163,8 @@ public static Optional<String> exchangeAPIKeyToJWT(String apiKeyId) {
             // Create a request to exchange API key to JWT.
             HttpPost exchangeRequest = new HttpPost(url.toURI());
             exchangeRequest.addHeader("Content-Type", ContentType.APPLICATION_JSON.toString());
+            exchangeRequest.addHeader(APIConstants.OAUTH_AGENT_TRACE_ID_HEADER,
```

Comment:
Let's add this to `exchangePATToJWT` as well.

Reply:
Added